spec/pul_metadata_services/pulfa_record_spec.rb

```diff
@@ -148,4 +140,21 @@
       end
     end
   end
+
+  context 'collection record' do
+  let(:record1_path) { File.join(fixture_path, 'C0652.xml')}
+
+    it "knows it's a collection" do
+      expect(subject.collection?).to be true
+    end
+
+    it 'grabs the unittitle' do
+      expected = ["Emir Rodriguez Monegal Papers"]
+      expect(subject.title).to eq expected
+    end
+
+    it "doesn't have data about being in a collection since it is a collection" do
+      expect(subject.collections).to be nil
```

Comment:
I used nil because that felt consistent with looking for an element that's not there. I would change it if someone had a reason it should be `[]`, though!

Reply:
@tpendragon updated. thanks!